SPECS/perl-CPAN-DistnameInfo/perl-CPAN-DistnameInfo.spec

```diff
@@ -0,0 +1,193 @@
+Name:           perl-CPAN-DistnameInfo
+Version:        0.12
+Release:        19%{?dist}
+Summary:        Extract distribution name and version from a distribution filename
+License:        GPL+ OR Artistic
+Group:          Development/Libraries
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://metacpan.org/release/CPAN-DistnameInfo
+Source0:        https://cpan.metacpan.org/authors/id/G/GB/GBARR/CPAN-DistnameInfo-%{version}.tar.gz
+BuildArch:      noarch
+BuildRequires:  make
+BuildRequires:  perl
+BuildRequires:  perl(version)
+BuildRequires:  perl(ExtUtils::MakeMaker) >= 6.76
+BuildRequires:  perl(strict)
+BuildRequires:  perl(warnings)
+# Tests:
```

Comment:
If these are test dependencies, can we gate their inclusion at build-time behind the %with_check variable we set during check builds?

Reply:
added %with_check for tests